sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/CosmosConfig.java

```diff
@@ -115,6 +161,17 @@ public CosmosConfigBuilder enableQueryMetrics(boolean queryMetricsEnabled) {
             return this;
         }
 
+        /**
+         * Set maxDegreeOfParallelism
+         *
+         * @param maxDegreeOfParallelism value to initialize
+         * @return CosmosConfigBuilder
+         */
+        public CosmosConfigBuilder setMaxDegreeOfParallelism(int maxDegreeOfParallelism) {
```

Comment:
Since this is a builder, we should not use `set` prefix for the API. Like other APIs, it should just be called as `maxDegreeOfParallelism(int maxDegreeOfParallelism)`

Reply:
Done.

Reply:
I still see the setter API names in some of the files. Wondering if we updated them across the board. Worth checking again!

Reply:
Apologies I didn't search the spring-test directory :)